src/main/java/com/turkcell/staj/entities/Transaction.java

```diff
@@ -0,0 +1,39 @@
+package com.turkcell.staj.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "packages")
+public class Transaction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+    @Column(name = "package_id")
+    private int packageId;
+    @Column(name = "user_id")
+    private int userId;
```

Comment:
`  @Column(name = "user_id")
    private int userId;`

bu satır FK ilişkilerinin tanımlanabilmesi için aşağıdaki satır ile değiştirilmelidir.
`    @ManyToOne()
@JoinColumn(name=user_id)
User user;`